project_liberation/project_liberation/settings/development.py

```diff
@@ -0,0 +1,12 @@
+from .base import *
+
+ENVIRONMENT = "development"
```

Comment:
What do we need this setting for?

Reply:
This seems to have been copied from our other projects. In the projects I initiate I always define this setting so that you can check what environment you're running in if you have to.

The concept came from Ruby on Rails which had a similar setting but to be honest I never actually had a real need to use it. I always prefer to add a new setting specific to the situation.

Reply:
I am just afraid that someone might be tempted to use this in application logic :)

Reply:
Then just remove it. I'm only explaining how it got here.

Reply:
Okay so I'm deleting this.